op-challenger/game/monitor_test.go

```diff
@@ -99,11 +100,21 @@ func setupMonitorTest(t *testing.T, allowedGames []common.Address) (*gameMonitor
 		i++
 		return i, nil
 	}
+	m := &mockMetricer{}
 	sched := &stubScheduler{}
-	monitor := newGameMonitor(logger, clock.SystemClock, source, sched, time.Duration(0), fetchBlockNum, allowedGames)
+	monitor := newGameMonitor(logger, m, clock.SystemClock, source, sched, time.Duration(0), fetchBlockNum, allowedGames)
 	return monitor, source, sched
 }
 
+type mockMetricer struct {
```

Comment:
This is going to get annoying to have to maintain as it will require updating every time we add a new metric.  Probably worth generating a mock with mockery to make it more maintainable, or at least put the mock somewhere reusable so we don't wind up maintaining multiple versions of it.

Actually I see it's not actually doing anything so currently could just be replaced with `NoopMetrics`.